frontend/package.json

```diff
@@ -1,36 +1,31 @@
 {
   "scripts": {
-    "start": "snowpack dev",
-    "build": "snowpack build",
-    "test": "jest",
+    "start": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "watch:re": "bsb -make-world -w",
```

Comment:
We have to run two commands `yarn watch:re` and `yarn start` separately when developing now. That's already what I was doing before so that's fine for me. If you prefer to only have one command though, we can use https://github.com/mysticatea/npm-run-all or https://github.com/kimmobrunfeldt/concurrently

Let me know if I should do that :)

Reply:
I've been running them separately too :)

Reply:
I find some of the names confusing. Specifically, I would imagine that `yarn build` would build the project (i.e., run bsb).

What do you think about these commands:

* `serve`: `vite`
* `build`: `bsb -make-world`
* `watch`: `bsb -make-world -w`
* `preview`: `vite preview`

This is not super important, I'm happy to stick to what you picked.

Reply:
I don't have any preferences, what your propose is fine for me. Building the project would require two commands though:

* `yarn bsb -make-world` to compile the `.res` files
* `yarn vite build` to actually produce

We can make the `build` command these two